### PR TITLE
vote & daily commands strings

### PR DESCRIPTION
**Pull request by FnrDev**
**Please describe the changes this PR makes and why it should be merged:**

**In this PR:**

### CODE REVIEW

File: en/bot.json [hunk 1]
```diff
@@ -521,5 +521,14 @@
...
         "color_relocate": ":rolling_eyes: - Please move my role above the selected color role.",
         "mute_text_description": "Mute a member from text channels so they cannot type",
-        "mute_voice_description": "Mute a member from voice channels so they cannot speak" 
+        "mute_voice_description": "Mute a member from voice channels so they cannot speak",
+	"vote_description": "Get the vote link and when you can vote for the bot",
+	"vote_success": "🎉 You are eligible to vote for the bot.",
+	"vote_wait_time": "😥 You can vote in **{0}**",
+	"vote_link_label": "Vote Link",
+	"vote_daily_tip": "**TIP:** You can get up to 3000 credits if you used daily",
+	"daily_success": "🎉 You are eligible to get daily for the bot.",
```
Comment (Anasqx) [inline en/bot.json:530]:
```suggestion
	"daily_success": "🎉 You are eligible to receive daily for the bot!",
```

File: en/bot.json [hunk 2]
```diff
@@ -521,5 +521,14 @@
...
+	"daily_link_label": "Get Daily Now",
+	"daily_wait_time": "You can claim daily in **{0}**",
```
Comment (Anasqx) [inline en/bot.json:532]:
```suggestion
	"daily_wait_time": "You can claim your daily in **{0}**",
```

File: en/bot.json [hunk 3]
```diff
@@ -521,5 +521,14 @@
...
         "mute_text_description": "Mute a member from text channels so they cannot type",
-        "mute_voice_description": "Mute a member from voice channels so they cannot speak" 
+        "mute_voice_description": "Mute a member from voice channels so they cannot speak",
+	"vote_description": "Get the vote link and when you can vote for the bot",
+	"vote_success": "🎉 You are eligible to vote for the bot.",
+	"vote_wait_time": "😥 You can vote in **{0}**",
+	"vote_link_label": "Vote Link",
+	"vote_daily_tip": "**TIP:** You can get up to 3000 credits if you used daily",
+	"daily_success": "🎉 You are eligible to get daily for the bot.",
+	"daily_link_label": "Get Daily Now",
```
Comment (Anasqx) [inline en/bot.json:531]:
```suggestion
	"daily_link_label": "Get Your Daily Reward Now!",
```

File: en/bot.json [hunk 4]
```diff
@@ -521,5 +521,14 @@
         "point_list_page_description": "Choose a points page number",
         "color_relocate": ":rolling_eyes: - Please move my role above the selected color role.",
         "mute_text_description": "Mute a member from text channels so they cannot type",
-        "mute_voice_description": "Mute a member from voice channels so they cannot speak" 
+        "mute_voice_description": "Mute a member from voice channels so they cannot speak",
+	"vote_description": "Get the vote link and when you can vote for the bot",
+	"vote_success": "🎉 You are eligible to vote for the bot.",
+	"vote_wait_time": "😥 You can vote in **{0}**",
+	"vote_link_label": "Vote Link",
+	"vote_daily_tip": "**TIP:** You can get up to 3000 credits if you used daily",
```
Comment (Anasqx) [inline en/bot.json:529]:
```suggestion
	"vote_daily_tip": "**TIP:** You can get up to 3000 credits if you claimed your daily!",
```

File: en/bot.json [hunk 5]
```diff
@@ -521,5 +521,14 @@
...
+	"daily_success": "🎉 You are eligible to get daily for the bot.",
+	"daily_link_label": "Get Daily Now",
+	"daily_wait_time": "You can claim daily in **{0}**",
+	"daily_vote_tip": "**TIP:** You can get up to 3000 credits if you vote for ProBot"
```
Comment (Anasqx) [inline en/bot.json:533]:
```suggestion
	"daily_vote_tip": "**TIP:** You can get up to 3000 credits if you vote for ProBot!"
```

**Review by Anasqx — CHANGES_REQUESTED** [review]

File: en/bot.json [hunk 6]
```diff
@@ -521,5 +521,14 @@
...
+        "mute_voice_description": "Mute a member from voice channels so they cannot speak",
+	"vote_description": "Get the vote link and when you can vote for the bot",
+	"vote_success": "🎉 You are eligible to vote for the bot.",
+	"vote_wait_time": "😥 You can vote in **{0}**",
+	"vote_link_label": "Vote Link",
+	"vote_daily_tip": "**TIP:** You can get up to 3000 credits if you claimed your daily!",
+	"daily_success": "🎉 You are eligible to receive your daily for the bot!",
+	"daily_link_label": "Get Your Daily Reward Now!",
+	"daily_wait_time": "You can claim your daily in **{0}**",
+	"daily_vote_tip": "**TIP:** You can get up to 3000 credits if you vote for ProBot!"
```
Comment (Anasqx) [inline en/bot.json:533]:
```suggestion
	"daily_vote_tip": "**TIP:** You can get up to 2600 credits if you vote for ProBot!"
```

File: en/bot.json [hunk 7]
```diff
@@ -521,5 +521,14 @@
         "point_list_page_description": "Choose a points page number",
         "color_relocate": ":rolling_eyes: - Please move my role above the selected color role.",
         "mute_text_description": "Mute a member from text channels so they cannot type",
-        "mute_voice_description": "Mute a member from voice channels so they cannot speak" 
+        "mute_voice_description": "Mute a member from voice channels so they cannot speak",
+	"vote_description": "Get the vote link and when you can vote for the bot",
+	"vote_success": "🎉 You are eligible to vote for the bot.",
```
Comment (Anasqx) [inline en/bot.json:526]:
```suggestion
	"vote_success": "🎉 You are eligible to vote for the bot!",
```

**Review by Anasqx — CHANGES_REQUESTED** [review]

File: en/bot.json (hunk 6 above)
Comment (Dramex) [reply en/bot.json:533]: This should be variable

Comment (FnrDev) [reply en/bot.json:533]: Done

**Pull request merged**